validation/src/test/scala/hmda/validation/engine/lar/validity/LarValidityEngineSpec.scala

```diff
@@ -1,17 +1,23 @@
 package hmda.validation.engine.lar.validity
 
-import hmda.model.fi.lar.LoanApplicationRegister
-import hmda.parser.fi.lar.LarGenerators
-import org.scalatest.prop.PropertyChecks
-import org.scalatest.{ MustMatchers, PropSpec }
+import java.io.File
+import hmda.parser.fi.lar.LarCsvParser
+import org.scalatest.{ MustMatchers, WordSpec }
 
-import scalaz.Success
+import scala.io.Source
 
-class LarValidityEngineSpec extends PropSpec with PropertyChecks
-    with MustMatchers with LarGenerators with LarValidityEngine {
+class LarValidityEngineSpec extends WordSpec with MustMatchers with LarValidityEngine {
 
-  property("Engine returns success for valid LAR") {
-    //TODO: test using a file of sample LARs, since the generator does not
-    //  produce records that pass all edit checks
+  "LAR Validity engine" must {
+    "pass validation on valid sample file" in {
```

Comment:
Thank you for adding a spec for this engine. It's valuable and will improve our test coverage.

But, I don't think this spec is sufficient to close #276. One of the requirements of that issue is "ensure that each check that should get run does get run."